src/browser_launchers/puppeteer_launcher.js

```diff
@@ -113,7 +115,11 @@ export class PuppeteerLauncher extends BrowserLauncher {
 
         if (this.stealth) {
             const { hideWebDriver, ...newStealthOptions } = this.stealthOptions;
-
+            log.deprecated(
+                'Puppeteer "stealth" and "stealthOptions" is deprecated'
```

Comment:
_are_ deprecated, also it looks like you are missing a dot and a space in here, this would result in `... is deprecatedYou should ...`